types/index.d.ts

```diff
@@ -322,17 +322,17 @@ declare namespace Realm {
         readonly prototype: Results<any>;
     };
 
-    class Credentials {
-        static emailPassword(email: string, password: string): Credentials;
-        static facebook(token: string): Credentials;
-        static apple(token: string): Credentials;
-        static gooogle(token: string): Credentials;
-        static anonymous(): Credentials;
-        static userAPIKey(key: string): Credentials;
-        static serverAPIKey(key: string): Credentials;
-        static custom(token: string): Credentials;
-        static function(token: string): Promise<Credentials>;
-    }
+    // class Credentials {
```

Comment:
Why is this commented out? How about adding a comment or TODO?

Reply:
fixed